lib/matplotlib/mlab.py

```diff
@@ -805,166 +806,60 @@ def csd(x, y, NFFT=None, Fs=None, detrend=None, window=None,
-                       sides=None):
-    """
-    Compute the magnitude (absolute value) of the frequency spectrum of
-    *x*.  Data is padded to a length of *pad_to* and the windowing function
-    *window* is applied to the signal.
-
-    Parameters
-    ----------
-    x : 1-D array or sequence
-        Array or sequence containing the data
-
-    %(Spectral)s
-
-    %(Single_Spectrum)s
-
-    Returns
-    -------
-    spectrum : 1-D array
-        The values for the magnitude spectrum (real valued)
-
-    freqs : 1-D array
-        The frequencies corresponding to the elements in *spectrum*
-
-    See Also
-    --------
-    psd
-        Returns the power spectral density.
-    complex_spectrum
-        This function returns the absolute value of `complex_spectrum`.
-    angle_spectrum
-        Returns the angles of the corresponding frequencies.
-    phase_spectrum
-        Returns the phase (unwrapped angle) of the corresponding frequencies.
-    specgram
-        Can return the complex spectrum of segments within the signal.
-    """
-    return _single_spectrum_helper(x=x, Fs=Fs, window=window, pad_to=pad_to,
-                                   sides=sides, mode='magnitude')
-
-
-@docstring.dedent_interpd
-def angle_spectrum(x, Fs=None, window=None, pad_to=None,
-                   sides=None):
-    """
-    Compute the angle of the frequency spectrum (wrapped phase spectrum) of
-    *x*.  Data is padded to a length of *pad_to* and the windowing function
-    *window* is applied to the signal.
-
-    Parameters
-    ----------
-    x : 1-D array or sequence
-        Array or sequence containing the data
-
-    %(Spectral)s
-
-    %(Single_Spectrum)s
-
-    Returns
-    -------
-    spectrum : 1-D array
-        The values for the angle spectrum in radians (real valued)
-
-    freqs : 1-D array
-        The frequencies corresponding to the elements in *spectrum*
-
-    See Also
-    --------
-    complex_spectrum
-        This function returns the angle value of `complex_spectrum`.
-    magnitude_spectrum
-        Returns the magnitudes of the corresponding frequencies.
-    phase_spectrum
-        Returns the phase (unwrapped angle) of the corresponding frequencies.
-    specgram
-        Can return the complex spectrum of segments within the signal.
-    """
-    return _single_spectrum_helper(x=x, Fs=Fs, window=window, pad_to=pad_to,
-                                   sides=sides, mode='angle')
-
-
-@docstring.dedent_interpd
-def phase_spectrum(x, Fs=None, window=None, pad_to=None,
-                   sides=None):
-    """
-    Compute the phase of the frequency spectrum (unwrapped angle spectrum) of
-    *x*.  Data is padded to a length of *pad_to* and the windowing function
-    *window* is applied to the signal.
-
-    Parameters
-    ----------
-    x : 1-D array or sequence
-        Array or sequence containing the data
-
-    %(Spectral)s
-
-    %(Single_Spectrum)s
+_single_spectrum_docs = """\
+Compute the {0} of *x*.
```

Comment:
```suggestion
Compute the {quantity} of *x*.
```
and call `format(quantity=)` to make it more readable.